syncwatcher.go

```diff
@@ -194,6 +197,11 @@ func init() {
 		os.Exit(0)
 	}
 
+	if accumulationInterval != 500 {
```

Comment:
Let's remove the conditional and always assign the parameter value. I don't like the repeated number here.